server/translations.json

```diff
@@ -25,6 +25,7 @@
     "pencil": "Stift",
     "private_board_description": "Du kannst ein <b>privates Whiteboard</b> mit einem zufälligen Namen erstellen, welches man nur mit seinem Link öffnen kann. Benutze dies wenn du private Informationen teilen möchtest.",
     "public_board_description": " Das <b>öffentliche Whiteboard</b> kann von jedem geöffnet werden. Es ein fröhliches unorganisiertes Chaos wo du zusammen mit anonymen Fremden malen kannst. Alles dort ist vergänglich.",
+    "recent_boards": "Neueste Boards",
```

Comment:
Where did you get these translations ? Do you speak all of these languages ? If not, then please be very careful with the translations, and just put the string in english in case of doubt.
Looking at your translations, they look like machine translations with words for "boards" that are incoherent with the rest of the text, and often completely unrelated to the concept of "whiteboard".